### PR TITLE
fixes #23112 - errata/ostree repo pages not showing content views

### PR DESCRIPTION
**Pull request by cfouant**

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/details/erratum-repositories.controller.js [hunk 1]
```diff
@@ -46,10 +49,20 @@ function ($scope, $q, Nutupane, Repository, Environment, ContentView, CurrentOrg
...
+                env = _.find(version.environment_ids, function(env_id) {
+                    return env_id === $scope.environmentFilter;
+                });
+
+                return !angular.isUndefined(env);
+            });
+
+            if (!angular.isUndefined(version)) {
+                params['content_view_version_id'] = version.id;
+            }
```
Comment (sjha4) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/details/erratum-repositories.controller.js]: @cfouant , @parthaa - 
Question:
Could we `delete params['content_view_version_id']` if the cv version is undefined? Changing environment filter (to an env without this cv) leads to no results, mostly cause the API call is made with the old cv filter and new env filter.

**Comment by cfouant** [comment]
@sjha4 - done - also updated the package repositories page to keep with uniformity of code. Thanks!

**Comment by jlsherrill** [comment]
Can you make sure that this passes 'grunt eslint'?   The pr this was based off did not, i've opened up a pr to fix that https://github.com/Katello/katello/pull/7318

**Comment by cfouant** [comment]
@jlsherrill - thanks for catching that, fixed!

**Pull request merged**